src/v/redpanda/application.cc

```diff
@@ -406,6 +406,14 @@ void application::setup_internal_metrics() {
           sm::description("Redpanda build information"),
           build_labels),
       });
+
+    _metrics.add_group(
+      "node:movements",
```

Comment:
Let's move it closer to `recovery_throttle.h` and use the `raft` namespace so that metrics reflect the code organization.

Reply:
fixed